arches/settings_utils.py

```diff
@@ -104,27 +144,45 @@ def build_templates_config(root_dir, debug, app_root=None, arches_applications=N
     ]
 
 
-def transmit_webpack_django_config(
-    root_dir, app_root, static_url, public_server_address, webpack_development_server_port, arches_applications=None
-):
-    arches_applications_paths = {}
+def transmit_webpack_django_config(**kwargs):
```

Comment:
Only by the time I ran into the issue with an ambiguous `import celery` statement (see note below -- checked with @aarongundel and he's run into this on deployments, too) -- did I start needing to feed additional settings. Had the idea to just feed them all in, see `**our_settings` below.

We lose some explicitness, but we hopefully gain some concision in the settings.py templates and avoid churn if we have to keep iterating on this in future versions.

Reply:
Agreed, in this case this is a better pattern 👍